seed/scraper.js

```diff
@@ -0,0 +1,180 @@
+  "https://www.houseplantsexpert.com/golden-barrel-cactus.html",
+  "https://www.houseplantsexpert.com/jade-plant.html",
+  "https://www.houseplantsexpert.com/jelly-beans-plant.html",
+  "https://www.houseplantsexpert.com/lithops-fulleri.html",
+  "https://www.houseplantsexpert.com/lithops-optica-rubra.html",
+  "https://www.houseplantsexpert.com/mother-in-laws-tongue.html",
+  "https://www.houseplantsexpert.com/mother-of-thousands-bryophyllum-daigremontianum.html",
+  "https://www.houseplantsexpert.com/orchid-cactus.html",
+  "https://www.houseplantsexpert.com/panda-plant.html",
+  "https://www.houseplantsexpert.com/peperomia-puteolata.html",
+  "https://www.houseplantsexpert.com/peruvian-apple-cactus.html",
+  "https://www.houseplantsexpert.com/peperomia-ferreyrae.html",
+  "https://www.houseplantsexpert.com/rat-tail-cactus.html",
+  "https://www.houseplantsexpert.com/tiger-jaws-faucaria-tigrina.html",
+  "https://www.houseplantsexpert.com/trailing-jade.html",
+  "https://www.houseplantsexpert.com/lithops-pseudotruncatella.html",
+  "https://www.houseplantsexpert.com/watermelon-peperomia.html",
+  "https://www.houseplantsexpert.com/zebra-haworthia.html",
+];
+
+async function asyncForEach(array, callback) {
+  for (let index = 0; index < array.length; index++) {
+    await callback(array[index], index, array);
+  }
+}
+
+puppeteer.launch({ headless: true }).then(async (browser) => {
+  const page = await browser.newPage();
+  await page.setViewport({ width: 1920, height: 926 });
+  const plants = [];
+
+  asyncForEach(arr, async (link) => {
+    await page.goto(link);
+
+    // get plant details
+    let plantData = await page.evaluate(() => {
+      let plant = {};
+      try {
+        plant.commonName = document
+          .querySelectorAll(".table")[0]
+          .querySelector("tbody > tr:nth-child(2) > td:nth-child(2)")
+          .innerText.split(".")[0];
+      } catch (exception) {}
+      try {
+        plant.scientificName = document.querySelector("h1").innerText;
+      } catch (exception) {}
+      try {
+        plant.image =
+          // document.querySelector("div.col-md-12 img").src ||
+          document.querySelector(".img-responsive").src;
+      } catch (exception) {}
+      try {
+        plant.light = document
+          .querySelectorAll(".table")[1]
+          .querySelector("tbody > tr:nth-child(2) > td:nth-child(2)").innerText;
+      } catch (exception) {}
+      try {
+        plant.watering = document
+          .querySelectorAll(".table")[1]
+          .querySelector("tbody > tr:nth-child(3) > td:nth-child(2)").innerText;
+      } catch (exception) {}
+      try {
+        plant.soil = document
+          .querySelectorAll(".table")[1]
+          .querySelector("tbody > tr:nth-child(4) > td:nth-child(2)").innerText;
+
+        // plant.watering: document.querySelector("div:nth-child(11) > div.row.row-code > div > table > tbody > tr:nth-child(3) > td:nth-child(2)")
+      } catch (exception) {}
```

Comment:
Do all the `try { plant.light }`, `try { plant.soil }` etc. need to be present for a plant to be included? or can a plant not have e.g. `soil` information?